classes/controller/action.php

```diff
@@ -0,0 +1,10 @@
+<?php 
+
+namespace mod_hsuforum\controller;
+
+interface action {
+	public function get_action($id);
+	public function set_action($id);
+	public function delete_action($id);
+	public function render_action($data);
```

Comment:
Noticed the calling functions reference $post. Can we update please to ensure consistency?

Reply:
interface should not care about naming, needs to be generic.

Reply:
Please explain why the above functions had id as a passing param vs render_action have data.